operations/tempo-mixin/out/tempo-operational.json

```diff
@@ -5025,6 +5025,142 @@
    "title": "Traces Missing Spans",
    "type": "bargauge"
   },
+  {
+   "datasource": "$ds",
+   "fieldConfig": {
+    "defaults": {
+     "color": {
+      "mode": "thresholds"
+     },
+     "mappings": [
+
+     ],
+     "thresholds": {
+      "mode": "absolute",
+      "steps": [
+       {
+        "color": "green",
+        "value": null
+       },
+       {
+        "color": "red",
+        "value": 0.050000000000000003
+       }
+      ]
+     },
+     "unit": "percentunit"
+    },
+    "overrides": [
+     {
+      "matcher": {
+       "id": "byName",
+       "options": "1800"
+      },
+      "properties": [
+       {
+        "id": "displayName",
+        "value": ".5h"
+       }
+      ]
+     },
+     {
+      "matcher": {
+       "id": "byName",
+       "options": "3600"
+      },
+      "properties": [
+       {
+        "id": "displayName",
+        "value": "1h"
+       }
+      ]
+     },
+     {
+      "matcher": {
+       "id": "byName",
+       "options": "10800"
+      },
+      "properties": [
+       {
+        "id": "displayName",
+        "value": "3h"
+       }
+      ]
+     },
+     {
+      "matcher": {
+       "id": "byName",
+       "options": "21600"
+      },
+      "properties": [
+       {
+        "id": "displayName",
+        "value": "6h"
+       }
+      ]
+     },
+     {
+      "matcher": {
+       "id": "byName",
+       "options": "43200"
+      },
+      "properties": [
+       {
+        "id": "displayName",
+        "value": "12h"
+       }
+      ]
+     },
+     {
+      "matcher": {
+       "id": "byName",
+       "options": "86400"
+      },
+      "properties": [
+       {
+        "id": "displayName",
+        "value": "24h"
+       }
+      ]
+     }
+    ]
+   },
+   "gridPos": {
+    "h": 9,
+    "w": 8,
+    "x": 16,
+    "y": 61
+   },
+   "id": 92,
+   "options": {
+    "displayMode": "gradient",
+    "orientation": "auto",
+    "reduceOptions": {
+     "calcs": [
+      "mean"
+     ],
+     "fields": "",
+     "values": false
+    },
+    "showUnfilled": true,
+    "text": {
+
+    }
+   },
+   "pluginVersion": "7.5.0-14141pre",
+   "targets": [
+    {
+     "expr": "sum(rate(tempo_vulture_trace_error_total{cluster=\"$cluster\", namespace=\"$namespace\", error=\"requestfailed\"}[1h])) / sum(rate(tempo_vulture_trace_total{cluster=\"$cluster\", namespace=\"$namespace\"}[1h]))",
```

Comment:
You can use [`$__rate_interval`](https://grafana.com/docs/grafana/next/datasources/prometheus/#using-__rate_interval-variable) instead of a fixed `1h`, which magically addresses a few [issues](https://grafana.com/blog/2020/09/28/new-in-grafana-7.2-__rate_interval-for-prometheus-rate-queries-that-just-work/) with rate functions intervals

Reply:
Thanks! I'll take a look at that 🙂

Reply:
Using `$__rate_interval` results in a totally different graph as a rate of `1h` smoothens out the result. Not sure which one is appropriate since the end result is single value and not a graph...

See: https://admin-ops-tools1.grafana.net/grafana/explore?left=%5B%22now-1h%22,%22now%22,%221-ops-tools1%22,%7B%22expr%22:%22sum(rate(tempo_vulture_trace_error_total%7Bcluster%3D%5C%22ops-us-east-0%5C%22,%20namespace%3D%5C%22tempo-ops%5C%22,%20error%3D%5C%22notfound%5C%22%7D%5B1h%5D))%20%2F%20sum(rate(tempo_vulture_trace_total%7Bcluster%3D%5C%22ops-us-east-0%5C%22,%20namespace%3D%5C%22tempo-ops%5C%22%7D%5B1h%5D))%22,%22interval%22:%22%22,%22datasource%22:%221-ops-tools1%22,%22exemplar%22:true,%22requestId%22:%22Q-db7bd5d9-842a-4115-948a-88eb2db83b1b-0Q-afef0543-5efc-4c03-9620-e8ee6ba07c8c-1A%22%7D,%7B%22exemplar%22:true,%22expr%22:%22sum(rate(tempo_vulture_trace_error_total%7Bcluster%3D%5C%22ops-us-east-0%5C%22,%20namespace%3D%5C%22tempo-ops%5C%22,%20error%3D%5C%22notfound%5C%22%7D%5B$__rate_interval%5D))%20%2F%20sum(rate(tempo_vulture_trace_total%7Bcluster%3D%5C%22ops-us-east-0%5C%22,%20namespace%3D%5C%22tempo-ops%5C%22%7D%5B$__rate_interval%5D))%22,%22requestId%22:%22Q-db7bd5d9-842a-4115-948a-88eb2db83b1b-0Q-afef0543-5efc-4c03-9620-e8ee6ba07c8c-1B%22,%22hide%22:false%7D%5D&orgId=1

Reply:
Prefer `$__rate_interval` unless we have a specific desire to see a rate as calculated over a longer time period. I think I set the vulture panels to use `1h` b/c I wanted any failure in vulture to be obvious and not disappear quickly. Honestly the visualization doesn't even make a ton of sense anymore. We used bar charts b/c we used to be able to see failures grouped by time range:

![image](https://user-images.githubusercontent.com/2272392/118015647-849be000-b322-11eb-93ab-2f566a6422ee.png)

It may be better to just go to a standard timeseries panel.

Reply:
Sure! I can figure out a better visualisation for all vulture metrics 👍